mrmustard/lab/abstract/transformation.py

```diff
@@ -383,3 +312,96 @@ def _repr_markdown_(self):
                 )
 
         return header + body
+
+
+class Unitary(Transformation):
+    def __init__(self, name: str, modes_in: list[int], modes_out: list[int]):
+        super().__init__(name=name, modes_in_ket=modes_in, modes_out_ket=modes_out)
+        self.is_unitary = True
+
+    def value(self, cutoff: int):
+        return self.U(cutoffs=[cutoff for _ in range(self.num_modes)])
```

Comment:
Apologies if this was here before. I don't think the value should force changing to fock representation)

Reply:
Yeah I forgot to change this. Thanks for spotting it!